brokerapi/brokers/models/service_broker.go

```diff
@@ -48,19 +47,6 @@ type ServiceAccountManager interface {
 	CreateAccountWithRoles(bindingID string, roles []string) (ServiceBindingCredentials, error)
 }
 
-type GCPCredentials struct {
```

Comment:
@josephlewis42 repeating our in-person conversation., The google credentials were being deserialized into this struct solely for access to the ProductId. So you removed it and simplified the logic for initializing from the environment. 

Is this correct?